app/templates/components/public/ticket-list.hbs

```diff
@@ -218,6 +218,12 @@
         {{/if}}
       </div>
     </div>
+    <div class="column sixteen wide right floated">
```

Comment:
This should only be shown to logged in users AFAIK

Reply:
If user click on this . Login modal opens